src/main/java/org/apache/commons/collections4/bloomfilter/IndexExtractor.java

```diff
@@ -31,15 +31,15 @@
  * @since 4.5
```

Comment:
Note that the since tags should state `4.5.0` but I'll do a pass before the next release, so you don't have to make this PR bigger ;-)